core/bindings.rs

```diff
@@ -7,6 +7,7 @@ use crate::isolate::SHARED_RESPONSE_BUF_SIZE;
 
 use rusty_v8 as v8;
 use v8::InIsolate;
+use v8::MapFnTo;
```

Comment:
// TODO don't expose internal MapFnTo

Reply:
Yeah that TODO exists twice now in rusty_v8, I think that's sufficient for the moment.